<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddMachineWinodwClass</class>
 <widget class="QWidget" name="AddMachineWinodwClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddMachineWinodw</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit_cpu"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_ram"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_rom"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_gpu"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_confirm">
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_cancal">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
